src/ucp/rndv/proto_rndv.h

```diff
@@ -116,7 +116,6 @@ void ucp_proto_rndv_rts_query(const ucp_proto_query_params_t *params,
 
 void ucp_proto_rndv_rts_abort(ucp_request_t *req, ucs_status_t status);
 
-
```

Comment:
why removed?

Reply:
By mistake while picking patch for pull request, I will restore